.vscode/launch.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
do you want to add this to your `~/.config/git/ignore`?

Reply:
Added...thanks.